src/hospital/PatientCollection.java

```diff
@@ -0,0 +1,46 @@
+package hospital;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ *
+ * @author maximdumont
+ */
+public class PatientCollection{
+    private List<Patient> patients = new ArrayList<>();
```

Comment:
We also need a way to look up patients by patient ID. What do you think about having a HashMap<Integer, Patient> with the key being the patient ID in addition to the ArrayList? 

 If you don't have time for changes Walt or I could do it.